.github/workflows/test-harness.yml

```diff
@@ -97,7 +97,7 @@ jobs:
       with:
         args: |
           bash -o pipefail -xc "cd /vector-test-harness \
-          && bin/compare -s vector -c default $(echo '${{ github.event.comment.body }}' | head -n 1 | sed 's|^/test ||') | tee \"$GITHUB_WORKSPACE/output\""
+          && bin/compare -s vector -c default -v nightly/latest -v ${{ steps.cloned-repo-info.outputs.test_harness_vector_version }} $(echo '${{ github.event.comment.body }}' | head -n 1 | sed 's|^/test ||') | tee \"$GITHUB_WORKSPACE/output\""
```

Comment:
@MOZGIII I'm curious how you think we should go about carrying over the `-c` flag to here?

Reply:
If we make `-c` flag settable via env vars, we'll be able to assign it by default to `default` just like at the test command above.

I'm also thinking about adding a command to the test harness repo that would be dedicated to handling the Github Action invocations. This would better decouple the test harness logic from the Github Action logic - I believe it'd be better if the latter is scoped to just triggering test harness, so that we offload unrelated logic from it as much as possible.

Reply:
> If we make `-c` flag settable via env vars, we'll be able to assign it by default to `default` just like at the test command above.

Btw, the same applies to `-s`.